src/Organic/Affiliate.php

```diff
@@ -25,16 +25,22 @@ public function register_scripts( $hook_suffix ) {
             // Gutenberg blocks are not supported
             return;
         }
+        // As more blocks are added, create a function like "register_widget_block_script".
+        $card_asset_file = include( plugin_dir_path( __DIR__ ) . 'blocks/affiliate/productCard/build/index.asset.php' );
+        // wp-scripts 5.0.0+ generates an index.asset.php file with dependencies. If the wp-scripts version is before
```

Comment:
Do we really need this legacy handling?

The `index.asset.php` is generated by our scripts that use the latest `wp-scripts` version. So, the `index.asset.php` will be generated and present in the plugin and accessible to read even if the plugin is installed on the older version of the WP

Reply:
Ah, good point. I'll get rid of this then. Thanks!